test/functionalTest/cases/2750_common_metrics/2750_incoming_errors.test

```diff
@@ -0,0 +1,519 @@
+                    "incomingTransactions": 1,
+                    "serviceTime": REGEX(.*)
+                },
+                "get/metrics": {
+                    "incomingTransactionResponseSize": 322,
+                    "incomingTransactions": 1,
+                    "serviceTime": REGEX(.*)
+                }
+            },
+            "sum": {
+                "incomingTransactionErrors": 1,
+                "incomingTransactionRequestSize": 24,
+                "incomingTransactionResponseSize": 474,
+                "incomingTransactions": 3,
+                "serviceTime": REGEX(.*)
+            }
+        },
+        "sum": {
+            "incomingTransactionErrors": 1,
+            "incomingTransactionRequestSize": 24,
+            "incomingTransactionResponseSize": 474,
+            "incomingTransactions": 3,
+            "serviceTime": REGEX(.*)
+        }
+    }
+}
+
+
+04. Send a request with an Invalid URL
+======================================
+HTTP/1.1 400 Bad Request
+Content-Length: 56
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "service not found",
+    "error": "BadRequest"
+}
+
+
+05. Send a request with an Invalid Service Path
+===============================================
+HTTP/1.1 400 Bad Request
+Content-Length: 111
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "Only /absolute/ Service Paths allowed [a service path must begin with /]",
+    "error": "BadRequest"
+}
+
+
+06. Send a request with an Invalid Content-Type
+===============================================
+HTTP/1.1 415 Unsupported Media Type
+Content-Length: 97
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "not supported content type: Invalid-Content-Type",
+    "error": "UnsupportedMediaType"
+}
+
+
+07. Send a request with invalid value for URI param /options/
+=============================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 76
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "Invalid value for URI param /options/",
+    "error": "BadRequest"
+}
+
+
+08. Send a request with a component-name too long in ServicePath
+================================================================
+HTTP/1.1 400 Bad Request
+Content-Length: 77
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "description": "component-name too long in ServicePath",
+    "error": "BadRequest"
+}
+
+
+09. Send a request with no acceptable mime-type in accept header
+================================================================
+HTTP/1.1 406 Not Acceptable
+Content-Length: 145
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
```

Comment:
I understand this NGSIv1-style error response is ok. If isn't, then a reference to the issue number should be include in a FIXME mark in the test step comment preamble.